src/components/SliderImage.tsx

```diff
@@ -56,9 +56,7 @@ export default function SliderImage() {
             key={slideIndex}
             onClick={() => goToSlide(slideIndex)}
             className="text-2xl cursor-pointer"
-          >
-            <RxDotFilled />
-          </div>
+          ></div>
```

Comment:
Not sure if this is the intended fix. Normally these dots are to show where in location the carousel is currently. May not be worthwhile. If you think that's the case, I would close this with a self closing div

Reply:
addressed.